indexd_test_utils2/__init__.py

```diff
@@ -0,0 +1,296 @@
+import hashlib
```

Comment:
Is there a more descriptive name that we can give this package rather than just tacking on 2 to the name? Or is the intention to eventually replace indexd_test_utils with this?

Reply:
I think we should eventually replace indexd_test_utils

Reply:
I agree, would you mind creating a cleanup ticket for that?